src/mscorlib/src/System/String.cs

```diff
@@ -573,10 +572,13 @@ public override bool Equals(Object obj)
         // Determines whether two strings match.
         [Pure]
         [ReliabilityContract(Consistency.WillNotCorruptState, Cer.MayFail)]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
@jamesqo I've just put this repo up so there could be some issues right out of the box.  Make sure to loop me in early and I can fix any issues.  :)